File: cst/tests/hello.f90
program main
use mpi
integer :: e, i, n
call mpi_init(e)
call mpi_comm_rank(mpi_comm_world, i, e)
call mpi_comm_size(mpi_comm_world, n, e)
print *, 'Process ', i, ' of ', n
!$omp parallel do schedule(static) private(i)
do i = 1, 100000000
end do
!$omp end parallel do
call mpi_finalize(e)
end program

File: Tests/hello.c
#include "mpi.h"
#include <stdio.h>

int main(int argc,char *argv[]) {
int i, n;
MPI_Init(&argc, &argv);
MPI_Comm_rank(MPI_COMM_WORLD, &i);
MPI_Comm_size(MPI_COMM_WORLD, &n);
fprintf(stdout,"Hello C process %d of %d\n", i, n);
fflush(stdout);
#pragma omp parallel for schedule(static) private(i)
for (i = 0; i < 100000000; i++) continue;
MPI_Finalize();
return 0;

}
